server/common/api/models.py

```diff
@@ -28,4 +28,12 @@ class Reservation(models.Model):
 	reservationID = models.AutoField(primary_key = True)
 	guestCount = models.IntegerField()
 	userEmail = models.ForeignKey(User, on_delete = models.CASCADE)
-	timeTableIdx = models.ForeignKey(TimeTable, on_delete = models.CASCADE)
\ No newline at end of file
+	timeTableIdx = models.ForeignKey(TimeTable, on_delete = models.CASCADE)
+
+class Image(models.Model):
+	imageIdx = models.AutoField(primary_key = True)
+	coverImage = models.ImageField(blank = True, null = True)
+	imageType = models.IntegerField(default = 1)
+						#Just class image for MVP
+						#classImage : 1 , placeImage : 2
```

Comment:
1) Can we indent these lines correctly?
2) Optional: There's a way to define enums and use that as field types in django.

https://hackernoon.com/using-enum-as-model-field-choice-in-django-92d8b97aaa63

Using int or string for predefined set of objects is bad, since it makes the system brittle (and difficult to test).

Reply:
enumerate imagetype